Source/WarpX.cpp

```diff
@@ -959,6 +959,14 @@ WarpX::ReadParameters ()
                     noy = particle_shape;
                     noz = particle_shape;
                 }
+#ifdef AMREX_USE_EB
+                if (particle_shape > 1)
```

Comment:
Please add here that embedded boundaries are also _enabled at runtime_, before raising this.

Does this affect EBs for ECT and ES solver? Otherwise, we should also check for that.

Reply:
@ax3l 
I added the runtime check, thanks a lot for the suggestion. I had to move the warning because where it used to be the EB data are not initialized yet. 

> Does this affect EBs for ECT and ES solver? Otherwise, we should also check for the exact field solver allow-list/warn-list here.

This affects EB with every possible solver (YEE, CKC, ECT, ES..) so I don't think we need any check for the solver here.